cellfinder/napari/detect/thread_worker.py

```diff
@@ -88,5 +95,8 @@ def classify_callback(batch: int) -> None:
             classify_callback=classify_callback,
             detect_finished_callback=detect_finished_callback,
         )
-        self.update_progress_bar.emit("Finished classification", 1, 1)
+        if not self.classification_inputs.skip_classification:
+            self.update_progress_bar.emit("Finished classification", 1, 1)
+        else:
+            self.update_progress_bar.emit("Finished detection", 1, 1)
```

Comment:
I am a bit surprised that code coverage thinks that L101 is not covered, btw. I thought it would be covered by `test_signals_detection_only` @FatmaElsharkawy do you understand what is happening?